src/lib/asset-service/service/generatedAssetData.json

```diff
@@ -80266,6 +80266,18 @@
     "explorerAddressLink": "https://arbiscan.io/address/",
     "explorerTxLink": "https://arbiscan.io/tx/"
   },
+  "eip155:42161/erc20:0xf929de51d91c77e42f5090069e0ad7a09e513c73": {
```

Comment:
Will have to figure out a way to consistently bring this guy in - this will get lost next `generatedAssetData.json` update

Reply:
Figured it out - see https://github.com/shapeshift/web/pull/6925/commits/2b659a8b7b3d425da1ff89bd665de48e221d2e02